src/openMVG/exif/sensor_width_database/sensor_width_camera_database.txt

```diff
@@ -656,6 +657,7 @@ Contax TVS Digital;7.11
 Contax N Digital;36
 DJI FC300S;6.16
 DJI FC300X;6.16
+#FC300X;6.16
```

Comment:
yes, ignore that. I was testing a script that will guess a focal length if it is not in the repo.